### PR TITLE
feat (MELB): "Songbird" additions

### PR DESCRIPTION
**Pull request by EidolonForgery**
Add: Shrike light CAS helicopter
Add: Magpie light transport helicopter
Add: Talon missiles + launcher
Add: Rake missiles + launcher
- Hides MELB variants that are now outdated

### CODE REVIEW

**Review by EidolonForgery — COMMENTED** [review]
I want this off my desk so bad. MFD is soul-draining. At least I got the grease pencil to fuck off

File: addons/patches/melb/CfgAmmo.inc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class cfgAmmo {
+	class M_AT;
+	class SWS_Dumbfire_Base : M_AT // Same as default hydras
+	{
+		cmimmunity = 1;
```
Comment (WitchFreya) [inline addons/patches/melb/CfgAmmo.inc:5]:
```suggestion
    ITEM_META(0);
		cmImmunity = 1;
```

File: addons/patches/melb/CfgAmmo.inc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	class SWS_Dumbfire_Base : M_AT // Same as default hydras
+	{
+    ITEM_META(0);
+		cmImmunity = 1;
+		whistleDist=24;
+		effectsMissileInit = "MissileDAR1";
+		muzzleEffect = "BIS_fnc_effectFiredRocket";
+	};
+	class SWS_M_Talon : SWS_Dumbfire_Base 
+	{
```
Comment (WitchFreya) [inline addons/patches/melb/CfgAmmo.inc:12]: Double check this + the others, I've run into issues omitting file extensions in some places before so for safety I just include them whether or not they're necessary.
```suggestion
		model = "melb\proxies\m229_fly.p3d";
```

File: addons/patches/melb/CfgAmmo.inc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		model = "melb\proxies\m229_fly";
```
Comment (WitchFreya) [inline addons/patches/melb/CfgAmmo.inc:13]:
```suggestion
		proxyShape = "melb\proxies\m229_proxy.p3d";
```

File: addons/patches/melb/CfgAmmo.inc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+		proxyShape = "melb\proxies\m229_proxy";
+		hit = 320;
+		indirectHit = 60;
+		indirectHitRange = 15;
+		suppressionRadiusHit = 20;
+	};
+	class SWS_M_Rake : SWS_Dumbfire_Base {
```
Comment (WitchFreya) [inline addons/patches/melb/CfgAmmo.inc:20]:
```suggestion
		model = "melb\proxies\m261_mpsm_fly.p3d";
```

Comment (WitchFreya) [inline addons/patches/melb/CfgAmmo.inc:20]:
```suggestion
		proxyShape = "melb\proxies\m261_mpsm_proxy.p3d";
```

File: addons/patches/melb/CfgAmmo.inc [hunk 5]
```diff
@@ -0,0 +1,34 @@
+class cfgAmmo {
```
Comment (WitchFreya) [inline addons/patches/melb/CfgAmmo.inc:1]:
```suggestion
class CfgAmmo {
```

File: addons/patches/melb/CfgMagazines.inc [hunk 6]
```diff
@@ -0,0 +1,142 @@
+class cfgMagazines
```
Comment (WitchFreya) [inline addons/patches/melb/CfgMagazines.inc:1]:
```suggestion
class CfgMagazines
```

**Review by WitchFreya — CHANGES_REQUESTED** [review]
Mostly looks good; I'd prefer it lives in two places though. The stuff that modifies MELB's base behavior should be in `addons\patches\melb` with 
```cpp
#define SUBCOMPONENT melb
#define SUBCOMPONENT_BEAUTIFIED Mission Enhanced Little Bird
```

while the stuff that adds the Songbird should be a subcomponent in `addons\air\heli_songbird` with 
```cpp
#define SUBCOMPONENT heli_sonbird
#define SUBCOMPONENT_BEAUTIFIED Songbird
```

**Comment by EidolonForgery** [comment]
> Mostly looks good; I'd prefer it lives in two places though. The stuff that modifies MELB's base behavior should be in `addons\patches\melb` with
> 
> ```c++
> #define SUBCOMPONENT melb
> #define SUBCOMPONENT_BEAUTIFIED Mission Enhanced Little Bird
> ```
> 
> while the stuff that adds the Songbird should be a subcomponent in `addons\air\heli_songbird` with
> 
> ```c++
> #define SUBCOMPONENT heli_sonbird
> #define SUBCOMPONENT_BEAUTIFIED Songbird
> ```

ouoeugouefoeughef

But, it will be done

**Comment by EidolonForgery** [comment]
Hold please, addressing one or two small fixes that I should have already checked.

**Review by WitchFreya — APPROVED** [review]

**Pull request merged**